articles/architecture-scenarios/b2c.md

```diff
@@ -1,108 +1,144 @@
 
-Any information captured during signup or progressive profiling can be stored in the Auth0 user profile. You can pass this information to the application through authentication tokens or get it using the Auth0 APIs.
+::: note
+You can find detailed guidance regarding specific functionality and use cases in our documentation or by speaking with your account representative or a member of our [Professional Services team](/services) here at Auth0.
+:::
 
-## Social login
+## Implementation overview
 
-Auth0 makes it easy to enable [login with social identity providers](/identityproviders#social). After a few simple configuration steps, users can log in with their Google, Facebook, LinkendIn, or other social accounts.
+Phase 1 focuses on integrating your application(s) with Auth0. During phase 1 you’ll tackle the 10 key stages to Go-Live, across 3 key milestones, and by tackling the main risk items will address the most critical paths associated with integration. At the end of this phase you’ll have working implementation that can be taken into production - or at the very least be provided as part of an early adopter or Beta program. 
 
-Social login removes potential barriers for users. Instead of giving a username and password, login becomes a single click. Once a user logs into a social provider their browser keeps a single sign-on session for them. This lets them access other applications that use the same provider without typing in their credentials again and again.
+Phase 1 consists of a number of workstreams, with a number of topics in each. The workstreams, topics and the order in which you address each is important, so we recommend you follow the guidance prescribed. That’s not to say you can’t or shouldn’t tackle work in parallel: User Provisioning and User Authentication, for example, could be tackled independently and at the same time, and these could both be tackled in parallel with your Branding efforts. In the majority of successful integration cases we’ve also found that different teams tackle different streams, and that this can provide significant benefit: your design and development team(s) would typically tackle implementation whilst at the same time your branding team would tackle Auth0 asset customization thus reducing overall time to market.
 
-Social login also provides consent screens. So if users give consent, your application can access select user profile attributes about the user.
+The diagram below provides an overall view of the planning associated with Phase 1. By Milestone 1, you will have completed major work required to integrate an application, will have addressed the most significant risk items, and will also be able to provide demonstrable functionality to stakeholders too. 
```

Comment:
There isn't a diagram at present, so do we want to remove this paragraph until there is a diagram?

Reply:
```suggestion
The diagram below provides an overall view of the planning associated with Phase 1. By Milestone 1, you will have completed major work required to integrate an application, will have addressed the most significant risk items, and will also be able to provide demonstrable functionality to stakeholders. 
```

Reply:
The diagram is ready to go, should we just include a JPEG of it for now and can worry about having a linkable version later?

Reply:
I need to make it a .png file. But, yes, send me the link to it and I'll include it.